packages/list-plugins/lib/tracking/byTracking.js

```diff
@@ -7,13 +7,8 @@ const _byTracking = ({ created = true, updated = true }) => ({
   ...byFieldOptions
 } = {}) => ({ fields = {}, hooks = {}, ...rest }) => {
   const relationshipOptions = {
-    type: Relationship,
+    type: AuthedRelationship,
     ref: 'User',
-    access: {
```

Comment:
I think we still want to set access control on this field. It's only meant to be updated by the plugin.

Reply:
Or are you expecting this to be handled in byFieldOptions? If so that would be a breaking change?

Reply:
It is; within [`AuthedRelationship`](https://www.keystonejs.com/keystonejs/fields-authed-relationship/#differences-from-relationship):

> _ Default access control set to `{ create: false, read: true, update: false }`_

Reply:
Ahhh no worries 👍

Reply:
I was looking at this and assumed no default: https://github.com/keystonejs/keystone/pull/2813/files#diff-4e323fb7b9e441b5bec81534f6bae71eR8

Reply:
That's the mechanism for _overwriting_ the defaults 👍

Reply:
🚀